Horizon/Horizon/Sources/Common/Domain/GetCoursesInteractor.swift

```diff
@@ -151,8 +151,8 @@ private extension CDDashboardCourse {
     func fetchNextUpModuleItems() -> AnyPublisher<DashboardCourse?, Never> {
```

Comment:
Since this function doesn't fetch anymore, let's rename it to `mapToDashboardCourse`.